### DISCUSSION TITLE
"import QR code backups from Google Authenticator"

### DISCUSSION THREAD

**Discussion by Mehuge** [2fast-team/2fast#175] [Q&A]
The Microsoft Store page suggests can import from google authenticator, I can't for the life of me find where this option is.

**Comment by jp-weber** [comment]
There is actually no extra point, but it is implemented in the standard functions. The QR code from the Google app can be read using the camera, or an image of the QR code can be scanned normally.

**Reply by Mehuge** [reply]
Thankyou so much. I would never have found that option. I thought that was for importing single accounts when setting them up.
